### PR TITLE
Ubuntu 17.10+ fails captive portal authentication with arubanetworks captive portals

### PR DESCRIPTION
**Pull request by xnox**
Signed-off-by: Dimitri John Ledkov <xnox@ubuntu.com>

### CODE REVIEW

File: 2018/DVE-2018-0001.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# DVE-2017-0001: securelogin.arubanetworks.com captive Wi-Fi portals fail to authorise with Ubuntu 17.10+ / systemd-resolved clients
```
Comment (shane-kerr) [inline 2018/DVE-2018-0001.md]: `DVE-2018-0001` probably.

**Review by shane-kerr — CHANGES_REQUESTED** [review]
Other than a typo in the DVE number assigned, this looks good to me.

File: 2018/DVE-2018-0001.md (hunk 1 above)
Comment (xnox) [reply 2018/DVE-2018-0001.md]: bah

**Review by shane-kerr — APPROVED** [review]
Looks good to me.

**Review by oerdnj — APPROVED** [review]

**Pull request merged**